docs/source/csv/standard.rst.part

```diff
@@ -110,48 +102,32 @@ The value of ``ApplicationRequestRejectionType`` must be one of the following:
 - missing-ssn
 - missing-state-id-number
 - missing-voter-signature
+- untracked
 - other
 
-.. _ApplicationRequestRejectionOtherType:
+.. _RequestRejectionOtherType:
 
-ApplicationRequestRejectionOtherType
-````````````````````````````````````
-**Data Type:** string
+RequestRejectionOtherType
+`````````````````````````
+**Data Type:** `string`_
 
 
 
-If :ref:`ApplicationRequestRejectionType` fails to cover all possible types of Application Rejection reasons, this field will list an alternate type.
+If :ref:`RequestStatusRejectionType` fails to cover all possible types of request rejection reasons, this field will list an alternate type.
 
 
 
-.. _ApplicationRequestStatusType:
+.. _RequestType:
 
-ApplicationRequestStatusType
-````````````````````````````
-**Data Type:** string
+RequestType
+```````````
+**Data Type:** `string`_
 
 *This field is required.*
 
-Specifies the current status of the application. Possible values are **accepted**, which denotes that the application was successfully processed; **cured** which means the application was supplemented with additional information to make it valid; **pending**, which signifies that application is still being processed; **rejected** which means the application was not successfully processed.
+Specifies the type of request that triggered the UOCAVA change. Possible values are **fpca**, which means a voter submitted a Federal Postcard Application; **fwab**, which denotes a voter returned a Federal Write-In Absentee Ballot prior to receiving or requesting a ballot through other means; **informal-request**, which signifies a voter requested a ballot through a less formal process, such as a letter or phone call; **nvra**, which means a voter used a National Voter Registration (Act) form; **state-application**, which denotes a state form triggered the activity; and **untracked**, which means the type of application is not tracked.
```

Comment:
Same question about the use of the term UOCAVA here as above.

Reply:
See above. Remove *UOCAVA*.